platform/jobs/edxEndToEndTestsJob.groovy

```diff
@@ -44,19 +44,41 @@ Map pipelineJob = [ name: 'edx-e2e-tests',
                     trigger: 'pipeline',
                     branch: '*/master',
                     description: 'Run end-to-end tests against GoCD deployments',
-                    courseNumber: 'AR-1000'
+                    courseNumber: 'AR-1000',
+                    testScript: 'jenkins/end_to_end_tests.sh',
+                    junitReportPath: 'reports/*.xml',
+                    junitScale: 1
                     ]
 
-// The edx-e2e-tests_pr job is run on every PR to the edx-e2e-tests repo. This
+// The edx-e2e-tests-pr job is run on every PR to the edx-e2e-tests repo. This
 // is used for development of the tests themselves
 Map prJob = [ name: 'edx-e2e-tests-pr',
               trigger: 'ghprb',
+              triggerPhrase: 'jenkins run e2e',
               branch: '${ghprbActualCommit}',
               description: 'Verify the quality of changes made to the end-to-end tests',
-              courseNumber: 'AR-1001'
+              courseNumber: 'AR-1001',
+              testScript: 'jenkins/end_to_end_tests.sh',
+              context: 'jenkins/e2e',
+              junitReportPath: 'reports/*.xml',
+              junitScale: 1
               ]
 
-List jobConfigs = [ pipelineJob, prJob ]
+// The microsites-staging-test-pr job is run on every PR to the edx-e2e-tests
+// repo. This is used for development of the tests themselves
+Map micrositesPrJob = [ name: 'microsites-staging-tests-pr',
+                        trigger: 'ghprb',
+                        triggerPhrase: 'jenkins run microsites',
+                        branch: '${ghprbActualCommit}',
+                        description: 'Verify the quality of changes made to the end-to-end tests',
+                        courseNumber: 'AR-1001',
+                        testScript: 'jenkins/white_label.sh',
+                        context: 'jenkins/microsites',
+                        junitReportPath: '*.xml,reports/*.xml',
+                        junitScale: 5
```

Comment:
Exactly. It allows you to define an 'amplification factor' to determine how many failures should fail a build